relentless/simulate/dilute.py

```diff
@@ -135,8 +135,9 @@ def __call__(self, sim):
                 r = sim.potentials.pair.r
                 u = sim.potentials.pair.energy((a,b))
                 f = sim.potentials.pair.force((a,b))
+                f[f==numpy.inf] = 0.                          # values omitted from integrand
```

Comment:
This will work, but your intention here is confusing since you are zeroing out the force. I would rather that you do something like in the `RelativeEntropy` class
https://github.com/mphowardlab/relentless/blob/02da640a9b7622762540aeb49bc0bd8d3b333edb/relentless/optimize/objective.py#L298-L303

Search for the first value of `u` that is finite (not `f`), then only integrate for r values from this point onward.